ui/src/asm_cleanup.rs

```diff
@@ -73,7 +74,8 @@ pub fn filter_asm(block: &str) -> String {
     let mut current_label: &str = "";
     let mut line_info: Vec<LineType> = Vec::new();
     let mut labels: HashSet<&str> = HashSet::new();
-    let mut opcode_operands: HashSet<&str> = HashSet::new();
```

Comment:
Why is this line changing? Is there extra whitespace?

Reply:
Yup, must have added an extra space. Removing.